app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(cp: str) -> None:
+    tokens = cp.split()
+    if len(tokens) != 3:
+        raise ValueError("Invalid command")
+    source_file, dest_file = tokens[1], tokens[2]
+    if source_file == dest_file:
+        return
```

Comment:
If command will equal to "cc" we also do copy?
https://github.com/mate-academy/py-copy-file/blob/master/checklist.md